src/app/shared/models/dot-action-bulk-result/dot-action-bulk-result.model.ts

```diff
@@ -8,6 +8,7 @@ export interface DotActionBulkResult {
     skippedCount: number;
     successCount: number;
     fails: DotBulkFailItem[];
+    action?: string;
```

Comment:
We need to update this to pass it where this is getting used: starting here: https://github.com/dotCMS/core-web/blob/master/src/app/portlets/dot-templates/dot-template-list/dot-template-list.component.ts#L486

Reply:
@fmontes I think that's the only place where it was missing.